code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -671,7 +671,7 @@
 
 /obj/item/reagent_containers/food/drinks/bottle/small/gingerbeer
 	name = "Ginger Beer"
-	desc = "A delicious non-alcoholic beverage enjoyed across Sol space."
+	desc = "A delicious non-alcoholic beverage enjoyed anywhere at anytime."
```

Comment:
???

Reply:
Are you not reading the descriptions.

Reply:
Do you enjoy ginger beer anywhere anytime? Sounds bad.

Reply:
Okay....- WHY DO YOU CARE SO MUCH IT'S LITERALLY- GUHHH
I'll just make it 'can be enjoyed anywhere.'

Reply:
Okay you kinda folded your point. I'm just gonna remove the enjoying point; who cares! **right??**

Reply:
desc = "A delicious non-alcoholic beverage."